### ISSUE TITLE
Rank() in subqueries sometimes does not pick up the alias

### ISSUE THREAD

**Issue by christopherpickering** [macbre/sql-metadata#204]
In this example query
```sql
select t.RANKED, t.RANKED_two, t.test, t.row_num from (SELECT
,RANK() OVER (PARTITION BY col_one ORDER BY col_two) RANKED
,RANK() OVER (PARTITION BY col_one ORDER BY col_two) as RANKED_two
,ROW_NUMBER() OVER (PARTITION BY col_one ORDER BY col_two) row_num
,ROW_NUMBER() OVER (PARTITION BY col_one ORDER BY col_two) as row_num_tow
, col_three as test
FROM nice_table) as t
where t.RANKED = 1
and t.RANKED_two = 2
```

The first rank has an alias of "RANKED", but does not have the "as" before it, so it is being skipped. The internals of the rank are treated as a subquery - which is cool - but "as" is not required with a rank alias.

**Comment by christopherpickering** [comment]
*Edited to remove comments about case sensitivity, which were not valid.

**Comment by collerek** [comment]
Ok, so this one was closed by #203 

Before the next release you can already install this one from the master branch from git:
`pip install git+https://github.com/macbre/sql-metadata.git`
